doc/variables.md

```diff
@@ -87,6 +87,8 @@ For more details see [scenario documentation](/doc/scenario.md).
 * `config` (`dict`, required): [instance configuration](/doc/instances.md);
 * `zone` (`string`): instance zone (available since
   [Cartridge 2.4.0](https://github.com/tarantool/cartridge/releases/tag/2.4.0));
+* `cartridge_extra_env` (`dict`): environment variables for instance service
```

Comment:
What about describing the difference between `null` and `{}`?

Reply:
Already....